README.md

```diff
@@ -56,6 +56,43 @@ Launches the prettier formatter. We recommend you to configure your IDE to run p
 Runs the Storybook in the development mode.  
 Open [http://localhost:6006](http://localhost:6006) to view it in your browser.
 
+## Local development environment
+
+A containerized environment, oriented to local development, is available to be run from the repository.
+
+This environment contains a dockerized instance of the Dataverse backend with its dependent services (database, mailserver, etc), as well as an npm development server running the SPA frontend (With code autoupdating).
```

Comment:
I just wanted to note that the autoupdating is working great!

I added "FOOBAR" to the Hello Dataverse page and as soon as I saved the file the text appeared, like this:

![Screen Shot 2023-05-09 at 12 06 28 PM](https://github.com/IQSS/dataverse-frontend/assets/21006/344cba93-ebbe-4252-84c4-44ee698cb9ef)

Here was the change:

```
$ git diff
diff --git a/src/sections/hello-dataverse/HelloDataverse.tsx b/src/sections/hello-dataverse/HelloDataverse.tsx
index 52091d7..a4bb4f5 100644
--- a/src/sections/hello-dataverse/HelloDataverse.tsx
+++ b/src/sections/hello-dataverse/HelloDataverse.tsx
@@ -8,6 +8,7 @@ export function HelloDataverse() {
   return (
     <section className={styles.container}>
       <h2 className={styles.title}>{t('title')}</h2>
+      <h1>FOOBAR</h1>
       <img src={logo} className={styles.logo} alt={t('altImage')} />
       <p>
         <Trans t={t} i18nKey="description" components={{ 1: <code /> }} />
```